kotlin/scene-layer/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
We can drop the constraint layout and just use the scene view as the single view for this sample. (I know the original java one had it, but we'll try to drop that pattern where possible)

Something like:

https://github.com/Esri/arcgis-runtime-samples-android/blob/master/kotlin/set-min-max-scale/src/main/res/layout/activity_main.xml